src/D2L.Bmx/Program.cs

```diff
@@ -215,6 +215,13 @@
 		next
 	) => {
 		await UpdateChecker.CheckForUpdatesAsync( configProvider.GetConfiguration() );
+		//Creating new Cache directory and removing old files.
+		if( !Directory.Exists( BmxPaths.CACHE_DIR ) ) {
+			Directory.CreateDirectory( BmxPaths.CACHE_DIR );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "awsCredsCache" ) );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "sessions" ) );
+			File.Delete( Path.Join( BmxPaths.BMX_DIR, "update_check" ) );
+		}
```

Comment:
Makes sense to handle exceptions gracefully here for the especially for the cleanup of the old files.

Reply:
I don't think the Directory or File functions error out even if the files don't exist

Reply:
Mhm, I was more concerned about the other exceptions. Especially for the cleanup part of this (i.e everything aside from CreateDirectory).
https://learn.microsoft.com/en-us/dotnet/api/system.io.file.delete?view=net-7.0#exceptions

Ideally we would like BMX to continue working if for w/e reason it can't actually delete those files.

Reply:
Gotcha! makes sense